src/view/aktivitet/Aktivitetskort.tsx

```diff
@@ -24,8 +24,7 @@ export function Aktivitetskort(props: PropTypes) {
         return null;
     }
 
-    const aktivitet =
-        aktiviteter.find(aktivitet => aktivitet.id === dialog!.aktivitetId) ||
+    const aktivitet = aktiviteter.find(aktivitet => aktivitet.id === dialog!.aktivitetId) ||
```

Comment:
her burde vi lage bedre gard !dialog || !dialog.aktivitetId
for å fjerne ! i find

Reply:
Det er bare å lage en pr på det :)